cmd/project/samples.go

```diff
@@ -60,7 +69,18 @@ func runSamplesCommand(clients *shared.ClientFactory, cmd *cobra.Command, args [
 	sampler := api.NewHTTPClient(api.HTTPClientOptions{
 		TotalTimeOut: 60 * time.Second,
 	})
-	selectedSample, err := PromptSampleSelection(ctx, clients, sampler)
+	samples, err := create.GetSampleRepos(sampler)
+	if err != nil {
+		return err
+	}
+	if samplesListFlag || !clients.IO.IsTTY() {
```

Comment:
👁️‍🗨️ note: We avoid attempting prompts in a non-interactive environment and instead list samples!

🗣️ note: The changes of #193 make this behavior more clear IMO since flag substitutes will then have matching prompts **or** the `create` command should be used.